docs/csharp/tuples.md

```diff
@@ -82,7 +82,7 @@ unnamed tuple:
 [!code-csharp[UnnamedTuple](../../samples/snippets/csharp/tuples/tuples/program.cs#01_UnNamedTuple "Unnamed tuple")]
 
 The tuple in the previous example was initialized using literal constants and
-won't have field names created using *Tuple field name projections* in C# 7.1.
+won't have element names created using *Tuple field name projections* in C# 7.1.
```

Comment:
You may want to do a scrub on the whole file for "fields". For instance, here you can say "Tuple element name projections".